src/main.zig

```diff
@@ -324,6 +324,8 @@ pub fn setNextKeypressListener(listener: ?*const fn(c_int, c_int, c_int) void) !
 fn escape() void {
 	if(game.world == null) return;
 	Window.setMouseGrabbed(!Window.grabbed);
+
+	gui.openWindow("pause_menu");
```

Comment:
This can be quite intrusive. The escape key has more functionalities than opening the pause menu.
It can also be used to just open the inventory (when it has been opened before)
`I` - open inventory
`Escape` - return to game
`Escape` - return to inventory
→ "Oh no the pause menu is now open, but I just wanted to return to the the inventory"
And even worse the pause menu also appears when you do this:
`I` - open inventory
`Escape` - return to game
`C` - open creative menu
→ "Oh no the pause menu is now open as well."

I think an approach like Terraria might be better. There pressing escape opens the inventory and other guis, but there is a small gear in the bottom right corner. Clicking on that opens the pause menu.
The gear would be a separate window with just a single (icon) button that opens the actual pause_menu. I'd recommend to set the following options for this window:
`.isHud = true,` → will always be open in the game
`.showTitleBar = false,`
`.hideIfMouseIsGrabbed = false,` → only shows the button when the cursor is free

Reply:
You're welcome. :)
Alright, before I resubmit this, I will:

1. Change the line endings back so that only functionally changed lines get changed.
2. Change the "windowlist" to use alphabetical order.
3. Replace all spaces with tabs.
4. Change the escape key back to its old functionality.
5. Add a small window that will allow you to open the pause menu.

I am going to leave the "Exit to main menu" and the "Invite player" buttons non-operational for right now.